doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -856,7 +856,12 @@ Locator tag is a small electronic device that can be attached to an object or a
 The locator tags can use different wireless technologies like GPS, Bluetooth LE or UWB for location tracking.
 It is even possible to combine multiple technologies in a single product to improve the user experience.
 
-The FMDN Accessory specification lists `Fast Pair Locator Tag Specific Guidelines`_ for the locator tag use case.
+The `Fast Pair Device Feature Requirements for Locator Tags`_ documentation defines the Fast Pair requirements for the locator tag use case.
+If your product is targeting the locator tag use cases, you must configure your application according to these requirements.
```

Comment:
`locator tag use case`?

Reply:
this was just a copy past of another sentence so I am going to align also that one